Cesium3DTiles/src/RasterOverlayTileProvider.cpp

```diff
@@ -524,6 +525,58 @@ RasterOverlayTileProvider::loadTileImageFromUrl(
           });
 }
 
+RasterOverlayTileProvider::LoadResult
+RasterOverlayTileProvider::processLoadedImage(
+    const CesiumGeometry::QuadtreeTileID& tileId,
+    LoadedRasterOverlayImage&& loadedImage) {
+
+  const auto& pPrepareRendererResources = this->getPrepareRendererResources();
+  const auto& pLogger = this->getLogger();
```

Comment:
We try to avoid passing pointers to `RasterOverlayTileProvider` and most other objects (including `this`) to a worker thread, because these objects are not thread safe (by design) and it's easy to accidentally misuse them from a thread.

In this case, it's probably fine (not a race condition). But by sticking to this rule we make any use of `this` or a pointer/reference to a non-thread-safe object a red flag, and that hopefully helps us avoid race conditions.

So this method should be static. At which point it could be a static global or private-namespaced method instead, which would have the additional benefit of letting us remove some implementation details from a public header.